DESIGN.md

```
@@ -1,329 +1,160 @@
+| **`primed`** | Substantive cold attempt on record; study unlocked | Learned, partially mastered |
+| **`drilled`** | Non-solid spaced reconstruction on record; return-worthy, never red | Failed, weak |
+| **`solidified`** | Solid spaced reconstruction on record (evidence, not mastery) | Mastered, cleared |
 
 ---
 
-## 3. The metacognitive happy path
-
-The binding seven-screen onboarding into any concept. Every concept-facing surface descends from this arc.
+## 3. State / data flow
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)